src/common/syncjournaldb.cpp

```diff
@@ -272,6 +272,11 @@ bool SyncJournalDb::sqlFail(const QString &log, const SqlQuery &query)
 bool SyncJournalDb::checkConnect()
 {
     if (_db.isOpen()) {
+        if (!QFile::exists(_dbFile)) {
+            qCWarning(lcDb) << "Database open, but file file" + _dbFile + " does not exist";
+            close();
+            return false;
+        }
```

Comment:
A comment explaining why this is here might make sense, because it looks pretty odd for somebody who does not know the history of this fix. This might easily look like a code left-over.